Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,266 @@
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Nightly              	| Unstable bits that are "bleeding edge". Users are not expected to use this channel often, however it is there for those situations when someone needs/wants a feature that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Preview 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Production          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Below table shows the mapping between the channels, branches and feeds for the Debian pacakage.
+
+| Channel    	| Branch    	| Debian feed 	| Debian package name 	|
+|------------	|-----------	|-------------	|---------------------	|
+| Nightly    	| master    	| Development 	| dotnet-nightly      	|
+| Preview    	| rel/<ver> 	| Development 	| dotnet              	|
+| Production 	| rel/<ver> 	| Production  	| dotnet              	|
+
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. The table below shows examples of that.
+
+| NuGet package version 	| Channel            	|
+|-----------------------	|--------------------	|
+| 1.0.0-dev-*           	| Dev channel        	|
+| 1.0.0-beta-*          	| Preview channel    	|
+| 1.0.0                 	| Production channel 	|
+
+# Funnels and discovery mechanisms for CLI bits
+There are multiple ways that we will funnel users towards the installers for the CLI:
+
+1. [Getting Started Page](https://aka.ms/dotnetcoregs)
+2. [Repo landing page](https://github.com/dotnet/cli/blob/rel/1.0.0/README.md)
+3. Package repositories for platforms (`apt-get`, `brew` etc.)
+4. IDEs and editors that integrate with CLI (e.g. Visual Studio, VS Code, Sublime etc.)
+
```

Comment:
This is not a funnel. Project build script is something that you build when you use the installer that we already funneled you to.